### ISSUE TITLE
project's broken windows need boarding up

### ISSUE THREAD

**Issue by aspiers** [jordansissel/xdotool#19]
As per [my post from October 2012](https://groups.google.com/d/msg/xdotool-users/ITghG7afJm0/CtAZvJ2O_ooJ), the xdotool project is currently in a confusing state for newcomers where it's hard to find out what the authoritative location for code and issue tracking is.  It would make a big difference if you took a few minutes to [board up the broken windows](http://pragprog.com/the-pragmatic-programmer/extracts/software-entropy), e.g.:
- fix the README to refer to this github repo, not to http://www.semicomplete.com/projects/xdotool/ which has been broken for a long time now.
- go through the 31 issues at https://code.google.com/p/semicomplete/issues/list, copy any which are still relevant to github, then close all of them with a comment explaining that development has moved to github
- fix http://www.semicomplete.com/projects/xdotool/

Thanks!

**Comment by eddy-geek** [comment]
Indeed, it was hard to find this page... The latest official package is also 2011.1, which makes one think it is not maintained anymore !
Do you want help with tagging & submitting new builds to distros ?

**Comment by jordansissel** [comment]
I've updated the project page on semicomplete.com to point here, now.

As for migrating the issues, I may not have energy to do that for a while; honestly, it's low priority, but I would welcome any help from others :)
